From 31b6bf2bfe7c98db4a038d1e76963917e6c4163d Mon Sep 17 00:00:00 2001
From: Zhiyuan Lv <zhiyuan.lv@intel.com>
Date: Mon, 23 Mar 2015 15:31:07 +0800
Subject: [PATCH 018/403] Change mask size to 32

Those masks are used for MMIO, which are 32-bit only.

Signed-off-by: Zhiyuan Lv <zhiyuan.lv@intel.com>
---
 drivers/gpu/drm/i915/vgt/reg.h |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/gpu/drm/i915/vgt/reg.h b/drivers/gpu/drm/i915/vgt/reg.h
index 0c39a2a..50dad26 100644
--- a/drivers/gpu/drm/i915/vgt/reg.h
+++ b/drivers/gpu/drm/i915/vgt/reg.h
@@ -113,12 +113,12 @@ static inline uint32_t __RING_REG(int32_t ring_id, uint32_t rcs_reg)
 #define RB_START(pdev, id)	(pdev->ring_mmio_base[id] + RB_OFFSET_START)
 #define RB_CTL(pdev, id)	(pdev->ring_mmio_base[id] + RB_OFFSET_CTL)
 
-#define RB_HEAD_OFF_MASK	((1UL << 21) - (1UL << 2))	/* bit 2 to 20 */
+#define RB_HEAD_OFF_MASK	((1U << 21) - (1U << 2))	/* bit 2 to 20 */
 #define RB_HEAD_OFF_SHIFT	2
-#define RB_TAIL_OFF_MASK	((1UL << 21) - (1UL << 3))	/* bit 2 to 20 */
+#define RB_TAIL_OFF_MASK	((1U << 21) - (1U << 3))	/* bit 3 to 20 */
 #define RB_TAIL_OFF_SHIFT	3
 
-#define RB_TAIL_SIZE_MASK	((1UL << 21) - (1UL << 12))	/* bit 12 to 20 */
+#define RB_TAIL_SIZE_MASK	((1U << 21) - (1U << 12))	/* bit 12 to 20 */
 #define _RING_CTL_BUF_SIZE(ctl)	(((ctl) & RB_TAIL_SIZE_MASK) + GTT_PAGE_SIZE)
 #define _RING_CTL_ENABLE	0x1	/* bit 0 */
 #define _RING_CTL_RB_WAIT	(1 << 11)
-- 
1.7.10.4

